src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const container = document.createElement('div');
+  const containerTitle = document.createElement('h2');
+  const containerDescr = document.createElement('p');
+
+  container.classList.add('notification', type);
+  container.style.top = `${posTop}px`;
+  container.style.right = `${posRight}px`;
+  containerTitle.classList.add('title');
+  containerTitle.innerText = title;
+  containerDescr.innerText = description;
+
+  container.appendChild(containerTitle);
+  container.appendChild(containerDescr);
+  document.body.appendChild(container);
+
+  setTimeout(() => {
+    document.body.removeChild(container);
+  }, 2000);
```

Comment:
Consider adding default values for the parameters of the pushNotification function. This can help prevent errors and make the function easier to use if some arguments are omitted.